assets/quick-add.css

```diff
@@ -94,6 +94,10 @@
   .quick-add-modal__content-info > * {
     max-height: 100%;
   }
+
+  quick-add-modal .product--mobile-columns .product__media-item {
+    width: calc(100% - 3rem - var(--grid-mobile-horizontal-spacing));
+  }
```

Comment:
The styling of the image isn't quite right still. It has a horizontal scrollbar that ideally wouldn't be there. But I think it's not a big issue and can be a follow up. 

Here is how I was seeing it: [video](https://screenshot.click/07-00-ixww6-gkdss.mp4)